br/pkg/lightning/mydump/parquet_parser.go

```diff
@@ -446,6 +449,13 @@ func setDatumByString(d *types.Datum, v string, meta *parquet.SchemaElement) {
 	if meta.LogicalType != nil && meta.LogicalType.DECIMAL != nil {
 		v = binaryToDecimalStr([]byte(v), int(meta.LogicalType.DECIMAL.Scale))
 	}
+	if meta.Type != nil && *meta.Type == parquet.Type_INT96 && len([]byte(v)) == 12 {
+		ts := int96ToTime([]byte(v))
+		ts = ts.UTC()
+		tsStr := ts.Format("2006-01-02 15:04:05.999999Z")
```

Comment:
why do we choose this layout?

Reply:
copy from https://github.com/pingcap/tidb/blob/ec9e43ed7e5a0b8e840da6e4dfe7695e1aac7355/br/pkg/lightning/mydump/parquet_parser.go#L512-L513

This layout is used when setting a UTC timestamp.